### ISSUE TITLE
Review Decision | Multiple decision documents tabs are stacking strangely.

### ISSUE THREAD

**Issue by kierachell** [department-of-veterans-affairs/caseflow#1273]
**AC**
1. Verify that decisions tabs show side by side. (Not stacked)
1. Verify that we can show up to 'x' number of tabs. (waiting on number)
1. Verify that the Tabs sit directly above the decision.
1. Verify that the type in the tabs is regular weight.

**Mocks:**
_Current:_

<img width="1099" alt="screen shot 2017-03-21 at 5 30 04 pm" src="https://cloud.githubusercontent.com/assets/18075411/24171772/258d5860-0e5c-11e7-902a-883ecb8ea6e4.png">

_Desired styling (from Style Guide):_
<img width="768" alt="screen shot 2017-03-23 at 8 43 27 am" src="https://cloud.githubusercontent.com/assets/4975959/24248204/e6637774-0fa4-11e7-8cae-2bf2e7def233.png">

**Comment by lakohl** [comment]
@cmgiven can you confirm how many decisions might appear in a 'multiple decision' worst case scenario?

**Comment by cmgiven** [comment]
Seven. You did ask for a worst case scenario. That happened in 2015.

But don't design for that. As long as this remains functional, we should almost never see more than four decisions, and that's pretty rare. Here's a table of how often decisions were cluster for the last three fiscal years. 1 indicates single cases, 2 indicates pairs of cases, and so on. There are only sixteen cases, or four groups of four, that would even hit the current max.

```
     1      2      3      4      7 
143688    952     27      4      1 
```

```r
decisions <- query("select BFCORLID, BFDDEC from BRIEFF where BFDC between '1' and '9' and BFAC in ('1', '3') and BFDDEC >= date '2013-10-01' and BFDDEC < date '2016-10-01'") %>%
  mutate(BFDDEC = as.Date(BFDDEC))

decisions$matches <- apply(decisions, 1, function(x) sum(decisions$BFCORLID == x['BFCORLID'] & abs(decisions$BFDDEC - as.Date(x['BFDDEC'])) <= 3))

summary(decisions$matches)
table(decisions$matches)
```

**Comment by cmgiven** [comment]
(Note that this is just looking at multiple decisions issued in a given time period. The other way this could occur is if multiple PDFs of the same decision were mistakenly uploaded. But I assume that 1. that is unlikely to happen five times, and 2. unlikely to cooccur with actual multiple decisions.)

**Comment by lakohl** [comment]
Thanks for the insight @cmgiven!

**Comment by lakohl** [comment]
low priority. closing
